sql/data-types/array-type.mdx

```diff
@@ -1,6 +1,6 @@
 ---
 title: "ARRAY type"
-description: "An array `T[]` is an ordered list of zero or more elements that share the same data type."
+description: "An array `T[]` is an ordered list of zero or more elements that share the same data type. RisingWave uses one-based indexing for arrays, meaning an array with n elements starts at `array[1]` and ends at `array[n]`."
```

Comment:
It is better to mention that this is a Postgres standard and RW just aligns with it:
```
In PostgreSQL standard, one-based indexing is used for arrays, meaning an array with n elements starts at `array[1]` and ends at `array[n]`." RisingWave uses one-based indexing as well to be align with the PostgreSQL.
```